lib/inferno/dsl/fhir_evaluator/evaluation_context.rb

```diff
@@ -0,0 +1,25 @@
+module Inferno
+  module DSL
+    module FHIREvaluation
+      # EvaluationContext is a wrapper class around the concepts needed to perform an evaluation:
+      # - The IG used as the basis for evaluation
+      # - The data being evaluated
+      # - A summary/characterization of the data
+      # - Evaluation results
+      class EvaluationContext
+        attr_reader :ig, :data, :results, :config
+
+        def initialize(the_ig, data, config)
```

Comment:
Modified

Reply:
It's because there's a rubocop complaint if we use just `ig` unfortunately:

```
lib/inferno/dsl/fhir_evaluator/evaluation_context.rb:12:24: 
C: Naming/MethodParameterName: Method parameter must be at least 3 characters long.
        def initialize(ig, data, config)
```

Reply:
Oh, right. It doesn't pass Rubocop with `ig`

Reply:
This is definitely a case where you should disable that rule for this line.

Reply:
Done